dipy/viz/ui.py

```diff
@@ -3230,8 +3231,10 @@ def _setup(self):
                                      vertical_justification="middle")
 
         # Add default events listener for this UI component.
-        self.textblock.on_left_mouse_button_clicked = self.left_button_clicked
-        self.background.on_left_mouse_button_clicked = self.left_button_clicked
+        self.add_callback(self.textblock.actor, "LeftButtonPressEvent",
```

Comment:
Is this really necessary, we should avoid directly adding callbacks to VtkProp whenever is possible.